internal/operands/common-templates/reconcile.go

```diff
@@ -106,9 +115,21 @@ func (c *commonTemplates) Reconcile(request *common.Request) ([]common.ResourceS
 	}
 
 	funcs = append(funcs, oldTemplateFuncs...)
-	funcs = append(funcs, reconcileTemplatesFuncs(request)...)
+	results, err := common.CollectResourceStatus(request, funcs...)
+	if err != nil {
+		return nil, err
+	}
 
-	return common.CollectResourceStatus(request, funcs...)
+	reconcileTemplatesResults, err := common.CollectResourceStatus(request, reconcileTemplatesFuncs(request)...)
+	if err != nil {
+		return nil, err
+	}
+	for _, r := range reconcileTemplatesResults {
+		if r.OperationResult == controllerutil.OperationResultUpdated {
+			CommonTemplatesRestored.Inc()
+		}
+	}
+	return append(results, reconcileTemplatesResults...), err
```

Comment:
Can you change the second argument to `nil`? It will be more clear that this `return` statement does not return any error.

Reply:
sure